spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/convert/ReadWriteConversions.java

```diff
@@ -27,4 +27,6 @@
 	<T> T convertOnRead(Object val, Class<?> targetType);
 
 	<T> T convertOnWrite(Object obj);
+
+	<T> T convertCollection(Object collection, Class<?> aClass);
```

Comment:
We shouldn't need this additional method. Read and write should already cover the purpose of this interface.